homeassistant/components/fireservicerota/sensor.py

```diff
@@ -0,0 +1,154 @@
+"""Sensor platform for FireServiceRota integration."""
+import logging
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import ATTR_ATTRIBUTION
+from homeassistant.helpers.dispatcher import async_dispatcher_connect
+from homeassistant.helpers.restore_state import RestoreEntity
+from homeassistant.helpers.typing import HomeAssistantType
+
+from .const import ATTRIBUTION, DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+async def async_setup_entry(
+    hass: HomeAssistantType, entry: ConfigEntry, async_add_entities
+) -> None:
+    """Set up FireServiceRota sensor based on a config entry."""
+    data = hass.data[DOMAIN][entry.entry_id]
+    entry_id = entry.entry_id
+    unique_id = entry.unique_id
+
+    async_add_entities(
+        [IncidentsSensor(data, entry_id, unique_id, "Incidents", "mdi:fire-truck")],
+        True,
+    )
+
+
+class IncidentsSensor(RestoreEntity):
+    """Representation of FireServiceRota incidents sensor."""
+
+    def __init__(self, data, entry_id, unique_id, name, icon):
+        """Initialize."""
+        self._data = data
```

Comment:
Maybe name this to something more obvious?
```suggestion
        self._coordinator = data
```